.circleci/config.yml

```diff
@@ -23,10 +23,79 @@ jobs:
           $job = start-job {dotnet run --project C:\Users\circleci\project\MockServer\MockServer}
           Start-Sleep 10
           .\Tests\doc-downloader.tests.ps1
+          .\Tests\drive-axle-doc-downloader.tests.ps1
       - run: |
           Write-Host (Split-Path -Path $PSScriptRoot)
           $ProgressPreference = 'SilentlyContinue';
           Compress-Archive -Path "C:\Users\circleci\project\README.txt", "C:\Users\circleci\project\configurations.json", "C:\Users\circleci\project\*.ps1" -DestinationPath document-downloader.zip
       - store_artifacts:
           path: document-downloader.zip
           destination: document-downloader.zip
+  build_powershell_six:
+      executor:
+        name: win/default
+        shell: powershell.exe
+      steps:
+        - checkout
+        - run: |
+            choco install powershell-core --version=6.2.4
+            pwsh.exe
+        - run: |
+            $module = "C:\Program Files\WindowsPowerShell\Modules\Pester"
+            takeown /F $module /A /R
+            icacls $module /reset
+            icacls $module /grant "*S-1-5-32-544:F" /inheritance:d /T
+            Remove-Item -Path $module -Recurse -Force -Confirm:$false
+        - run: |
+            Install-Module -Name Pester -Force  
+            Get-Host | Select-Object Version 
+            dotnet build C:\Users\circleci\project\MockServer\MockServer 
+            $job = start-job {dotnet run --project C:\Users\circleci\project\MockServer\MockServer}
+            Start-Sleep 10
+            .\Tests\doc-downloader.tests.ps1
+            .\Tests\drive-axle-doc-downloader.tests.ps1
```

Comment:
How do we know the tests are actually using the installed version of PowerShell, as opposed to picking up some version installed by the system? (Genuine question, the answer might be as simple as "there is no default version of PowerShell, if we don't install one, these all fail!)

Reply:
Yea so based on the results, just ran, when we install a version higher than the defualt 7.2.5, it will use the higher version (7.3.0 in build_powershell_seven's case), but when the version is lower than the default, 7.2.5 is still used (you can see in the `choco install powershell-core --version=6.2.4 pwsh.exe` command in the build_powershell_six build; it uses 7.2.5)

I'll need to figure out how to use the correct version.